apps/vmq_server/priv/vmq_server.schema

```diff
@@ -31,6 +31,13 @@
                                                                                                 {datatype, flag}
                                                                                                ]}.
 
+%% @doc the vernemq admin api key
+{mapping, "api_key", "vmq_server.api_key", [
+                                                 {commented, "Api-Key"},
+                                                 {default, undefined},
+                                                 {datatype, string}
+                                                ]}.
+
```

Comment:
The config option should probably have the same semantics as the `vmq-admin api-key add` which replicates the api key to all nodes in the cluster. Otherwise the other commands like `vmq-admin api-key delete` would work differently on config file keys and other keys which I think would be confusing. So to set the key `vmq_http_mgmt_api:add_api_key/` should be used.

Reply:
No worries. Ok, will take a look later. Regarding "differently on config file keys and other keys"  I have thought about this as well, but the proposed changes would lead to a lock out option, when the user deletes the key. Furthermore on cluster/instance reboot the key would be added again.